types/system-task/index.d.ts

```diff
@@ -2,25 +2,59 @@
 // Project: https://github.com/leocwlam/system-task
 // Definitions by: Leo Lam <https://github.com/leocwlam>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.1
 
-/// <reference types="node" />
+declare class SystemTask {
+    type: string;
 
-export = SystemTask;
+    constructor(taskType?: string, isAsyncProcess?: boolean, logMethod?: any);
 
-declare class SystemTask {
-  type: string;
-  constructor(taskType: string, isAsyncProcess?: boolean, logMethod?: any);
-  log(type: string, message: string, detail?: any): void;
-  insertPreprocessItemsHandler(task: SystemTask): Promise<any>;
-  preprocessHandler(task: SystemTask, preProcessItem: any): Promise<any>;
-  processHandler(task: SystemTask, processItem: any): Promise<any>;
-  cleanupHandler(task: SystemTask, cleanupItems: any[]): Promise<any>;
-  isValidProcess(): void;
-  start(): void;
+    /**
+     * @async
+     */
+    log(type: string, message: string, detail?: any): void;
+
+    /**
+     * @async
+     */
+    insertPreprocessItemsHandler(task: SystemTask): Promise<any>;
+
+    /**
+     * @async
+     */
+    preprocessHandler(task: SystemTask, preProcessItem: any): Promise<any>;
+
+    /**
+     * @async
+     */
+    processHandler(task: SystemTask, processItem: any): Promise<any>;
+
+    /**
+     * @async
+     */
+    cleanupHandler(task: SystemTask, cleanupItems: any[]): Promise<any>;
+
+    isValidProcess(): void;
+
+    /**
+     * @async
+     */
+    start(): void;
 }
 
+// tslint:disable-next-line:prefer-declare-function
+declare const asyncProcess: (items: any[], executeAsyncCall: any, task: SystemTask, errors: any[]) => Promise<any>;
+// tslint:disable-next-line:prefer-declare-function
+declare const syncProcess: (items: any[], executeSyncCall: any, task: SystemTask, errors: any[]) => Promise<any>;
+
 declare namespace SystemTask {
-  function asyncProcess(items: any[], executeAsyncCall: any, task: SystemTask, errors: any[]): Promise<any>;
-  function syncProcess(items: any[], executeSyncCall: any, task: SystemTask, errors: any[]): Promise<any>;
+    /**
+     * @async
+     */
+    const SyncProcess: typeof syncProcess;
```

Comment:
The lowercase names were just mistakes?

Reply:
No, this is OK IMO. Package defines those on export, `syncProcess`, exported as `SyncProcess`, thx!